AirBench/Scripts/Home.js

```diff
@@ -171,11 +170,23 @@
                     }
                 }
                 
-                //console.log(count);
             }
         }
         
     }
+    function description(){
+        let descriptions = document.getElementsByClassName('description');
+        for (let i= 0; i < descriptions.length; i++){
+            let d = descriptions[i].innerText;
+            let dArray = d.split(' ');
+            if (dArray.length > 10){
+                d = dArray.slice(0,10).join(' ');
+                d += '...';
+                descriptions[i].innerText = d;
+            }
+        }
+    }
```

Comment:
Interesting approach, but why not include a `ShortDescription` property on your `Bench` server-side model that returns the required short description string in order to keep from having to send any extra data to the client than what's necessary?